content/en/docs/ops/ambient/helm-installation/index.md

```diff
@@ -0,0 +1,90 @@
+---
+title: Installing Ambient Mesh with Helm
+description: How to install Ambient Mesh with Helm.
+weight: 4
+owner: istio/wg-environments-maintainers
+test: n/a
+---
+
+This guide shows you how to install Ambient Mesh with Helm.
+Besides the demo in [Getting Started with Ambient Mesh](/docs/ops/ambient/getting-started/),
+we **encourage** you to follow this guide to install Ambient Mesh.
+Helm helps you manage components separately, and you can easily upgrade the components to the latest version.
+
+## Prerequisites
+
+1. Perform any necessary [platform-specific setup](/docs/setup/platform-setup/).
+
+1. Check the [Requirements for Pods and Services](/docs/ops/deployment/requirements/).
+
+1. [Install the Helm client](https://helm.sh/docs/intro/install/), version 3.6 or above.
+
+1. Configure the Helm repository:
+
+{{< text bash >}}
+$ helm repo add istio https://istio-release.storage.googleapis.com/charts
+$ helm repo update
+{{< /text >}}
```

Comment:
```suggestion
    {{< text bash >}}
    $ helm repo add istio https://istio-release.storage.googleapis.com/charts
    $ helm repo update
    {{< /text >}}
```

Reply:
Indenting keeps it "under" the ordered list item instead of breaking out of the list, and allows for further additions without needing to worry about numbering in the future.